flocker/acceptance/test_postgres.py

```diff
@@ -40,6 +40,12 @@
         Port(internal_port=POSTGRES_INTERNAL_PORT,
              external_port=POSTGRES_EXTERNAL_PORT),
         ]),
+    environment=frozenset({
+        'LANG': 'en_US.utf8',
+        'PGDATA': '/var/lib/postgresql/data',
+        'PG_VERSION': '9.4.1-1.pgdg70+1',
```

Comment:
1. You can and should just pass a dictionary these days.
2. Why was this change made? If the expectation s that `PG_VERSION` will be set and therefore this is required for comparisons to work, that seems super-fragile. This may also indicate a weakness in our comparison logic that needs to be fixed in this branch - if we're saying "oh this container is no good because it has extra environment variables that weren't configured" then that's a problem.